evaluators/text/BleuEvaluator/tests/test_bleuevaluator.py

```diff
@@ -0,0 +1,16 @@
+from .. import BleuEvaluator
+import pytest
+
+
+def test_bleu_evaluator():
+    reference = [ ['All', 'cats', 'are', 'beautiful'], ['All', 'cats', 'are', 'cute'] ]
```

Comment:
It is weird, I think what the encoder should see is only `strings` and not `arrays of strings`. This is gonna be used to evaluate texts from crafters. Another thing is wether inside we may need to cut it.

Reply:
To be clear, this will compare only one string to another one? in this example, the reference (desired) will be a list of strings or will also be only one string?

Reply:
As per my understanding we should have :

`BleuEvaluator.evaluate('All cats are beautiful', 'All cats are cute')`

Reply:
This will compare one-by-one a document `jina_pb2.Document.text` vs a  groudntruth `jina_pb2.Document.text`

Reply:
Also try to add tests with edge cases, this can serve as documentation on how things work. Like what happens with `lower` and `upper` cases, what happens if the `desired` is empty or if the `actual` is empty ...

Reply:
Yeah sure, it has still many tests missing, this was just the first draft to check everything was in order

Reply:
yes, just reminder ...